.github/workflows/code-quality-main.yaml

```diff
@@ -0,0 +1,16 @@
+# Same as `code-quality-pr.yaml.yaml` but triggered on commit
```

Comment:
```suggestion
# Same as `code-quality-pr.yaml` but triggered on commit
```

Reply:
done in https://github.com/ashleve/lightning-hydra-template/pull/364/commits/479def86850bfe89ca07d1e7a30eb83c5a5f640f